radio-service-ui/src/pwa_utils/useShouldShowPrompt.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+import moment from 'moment';
+
+const getInstallPromptLastSeenAt = (promptName) => localStorage.getItem(promptName);
+
+const setInstallPromptSeenToday = (promptName) => {
+  const today = moment().toISOString();
+  localStorage.setItem(promptName, today);
+};
+
+function getUserShouldBePromptedToInstall(promptName, daysToWaitBeforePromptingAgain) {
+  const lastPrompt = moment(getInstallPromptLastSeenAt(promptName));
+  const daysSinceLastPrompt = moment().diff(lastPrompt, 'days');
+  return isNaN(daysSinceLastPrompt) || daysSinceLastPrompt > daysToWaitBeforePromptingAgain;
+}
+
+const useShouldShowPrompt = (promptName, daysToWaitBeforePromptingAgain = 30) => {
```

Comment:
30 days to wait before another prompt??
make it 6 hours....
in the future we have to add "never ask again" feature.